core/domain/html_cleaner.py

```diff
@@ -17,15 +17,22 @@
 """HTML sanitizing service."""
 
 import HTMLParser
+import cStringIO
 import json
 import logging
+import urllib
 import urlparse
 
 import bleach
 import bs4
 from core.domain import rte_component_registry
+from core.platform import models
 import feconf
 
+from PIL import Image
+
```

Comment:
Looks like this file is importing from oppia/third_party. Can you try removing the entries in `install_third_party.sh`and see if the back-end tests still pass?

Reply:
The backend tests would pass anyway because they do not involve usage of PIL. We have a mock function instead of the actual function, because the original function requires an image which must be fetched from some url.

Reply:
PTAL https://travis-ci.org/oppia/oppia/jobs/410598379#L747

Reply:
Okay so you want me to remove it from third_party.sh and keep the one that installs it from manifest.json ?

Reply:
Are you getting the same error locally or this is Travis only?

Reply:
How did you install PIL on your machine? `pip install`?
If you did a global `pip install` on your machine, I suggest you add that command on to .travis.yml as a quick work around.
Like  my recommendation in  #5335, I suggest you keep trying until you can replicate Travis locally.

Reply:
Yes i installed it globally using pip install. Even If i add this to
.travis.yml won't the backend tests i run locally look for PIL in my
machine?

One Clarification ---- , okay i would replicate by adding this to the
.travis.yaml but how do i verify it then?

Reply:
Changes to .travis.yml is only a quick hack to see if that is indeed Travis only issue.
Can you try uninstalling all local copy of Pillow on your machine until you are unable to import it like Travis?

Reply:
I unistalled all local copy of Pillow on my machine and yes the linter script gave the same error as on travis that `unable to import PIL`
![screenshot from 2018-08-01 21-52-30](https://user-images.githubusercontent.com/23234392/43534881-0d9c7ca6-95d6-11e8-97cf-3a7ec537a0cb.png)


This was the error when i ran `bash ./scripts/run_backend_tests.sh --test_target=core.domain.html_cleaner_test.py `

"It says this is because of an import error"
![1](https://user-images.githubusercontent.com/23234392/43534950-47df5488-95d6-11e8-85df-1bf95c03c905.png)

So after making changes to the travis, shall I push the commits, so that I could see the changes.
I am unable to get your point of making the changes to .travis.yaml, how can I test that locally ?

Reply:
OK great. Thank you. From your local, how do you go about fixing the lint error? Do you run the current `install_third_party.sh`?

Reply:
No that would just add it to the oppia_tools. The error is fixed by installing it on my machine.